src/main/resources/soracom-endorse-version

```diff
@@ -1 +1 @@
-Version: 0.1.0
\ No newline at end of file
+version = '00.03.02'
```

Comment:
Can we use "Version: 0.2.0" instead of version = '00.03.02'  ?

Reply:
Right. I changed the version number.